packages/graph/deployment/graph/migrate_and_run.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+set -eux
+
+cd /migrations
+
+HASH_GRAPH_PG_MIGRATION_URL=postgres://${HASH_GRAPH_USER}:${HASH_GRAPH_PASSWORD}@${HASH_GRAPH_HOST}:${HASH_GRAPH_API_PORT}/${HASH_GRAPH_DATABASE} yarn graph:migrate up
```

Comment:
I think we should use the root credentials to run the migrations on the database, incase we restrict permissions of the `HASH_GRAPH_USER` in a way that inhibits its ability to create/update tables/columns.
```suggestion
HASH_GRAPH_PG_MIGRATION_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${HASH_GRAPH_HOST}:${HASH_GRAPH_API_PORT}/${HASH_GRAPH_DATABASE} yarn graph:migrate up
```

Because the migrations are currently run in the `graph` container, that would mean we have to (temporarily) pass the root postgres credentials to that container (i.e `POSTGRES_USER` and `POSTGRES_PASSWORD`)

Reply:
I'm wondering the same thing, it seems the `init-user-db.sh` script is not setting up permissions as intended

Reply:
Fixed in ae6fa88fbcf64bb62e3eaa6bec81849216fb2564